Tests/TestingTests/ABIEntryPointTests.swift

```diff
@@ -20,6 +20,35 @@ private import _TestingInternals
 @Suite("ABI entry point tests")
 struct ABIEntryPointTests {
   @Test func v0() async throws {
+    var arguments = __CommandLineArguments_v0()
```

Comment:
Were the changes in this file necessary? Is there an ABI break? Will this conflict with my draft PR?

Reply:
None of the changes here were in response to API or ABI changes, I was only wanting to refactor things to make it easier to write a second test of the entry point without needing to duplicate a bunch of set up code

Reply:
The irony being I'm pitching a different signature for the entry point so this code ends up dead. 🥲